atomate/vasp/workflows/base/library/SCAN_optimization.yaml

```diff
@@ -1,4 +1,16 @@
 # SCAN structure optimization
 # Author: Ryan Kingsbury (RKingsbury@lbl.gov)
 fireworks:
-- fw: atomate.vasp.fireworks.core.ScanOptimizeFW
\ No newline at end of file
+- fw: atomate.vasp.fireworks.core.ScanOptimizeFW
+  params:
+    name: "PBESol pre-optimization for SCAN"
+    vasptodb_kwargs:
+      task_fields_to_push: {"gga_bandgap": "calcs_reversed.0.output.bandgap"}
+      parse_bader: False
+- fw: atomate.vasp.fireworks.core.ScanOptimizeFW
+  params:
+    name: "SCAN structure optimization"
+    parents: 0
+    prev_calc_loc: True
+    vasptodb_kwargs:
+      parse_bader: False
```

Comment:
@utf @mkhorton @shyamd do you have any feedback on the respective FW names here?

Reply:
I don't know that "for SCAN" is necessary, I would change the first firework to be "PBEsol structure optimization" (lower-case s)... under the logic that this way it states what it is, rather than what it's for, because there's zero SCAN specific stuff in the first firework.